src/components/Card/Card.tsx

```diff
@@ -415,7 +415,10 @@ export default class Card extends React.Component<CardProps, {
               title={this.props.type === "snippet" ? this.props.item.title : this.props.item.manifest?.name}
               className="main-cardHeader-link"
               dir="auto"
-              href={`https://github.com/${this.props.item.user}/${this.props.item.repo}`}
+              href={this.props.type !== "snippet"
+                ? `https://github.com/${this.props.item.user}/${this.props.item.repo}`
+                : "https://github.com/spicetify/spicetify-marketplace/blob/main/resources/snippets.json"
```

Comment:
You can use the SNIPPETS_URL constant rather than hard coding the URL in here

Reply:
SNIPPETS_URL returns a Raw GirHub link, I'm trying to use the Web UI one.